rotkehlchen/types.py

```diff
@@ -1160,6 +1160,7 @@ class CacheType(Enum):
     CURVE_LENDING_VAULT_AMM = auto()
     CURVE_LENDING_VAULT_COLLATERAL_TOKEN = auto()
     CURVE_LENDING_VAULT_BORROWED_TOKEN = auto()
+    AURA_POOLS = auto()  # stores the count of pools in db + chain
```

Comment:
specify here what chain means (once you change it). Chain id or chain name. I would go for chain id (stringified)